src/AuthenticatedAPIClient/AccessToken.js

```diff
@@ -84,6 +84,9 @@ export default class AccessToken {
     try {
       decodedAccessToken = decodeJwtCookie(this.cookieName);
     } catch (e) {
+      // Swallow this error and continue because we
```

Comment:
We aren't swallowing.
Maybe: `Log unexpected error and continue with attempt to refresh it.`

Reply:
Done